src/models.py

```diff
@@ -0,0 +1,158 @@
+"""Defines models for the whole project: Photo, Rover and Camera."""
```

Comment:
feels like you shouldbe installing the other app to get these models.  Rather than duplicating the file over here.